Dockerfile

```diff
@@ -5,8 +5,6 @@ RUN apk add --update ca-certificates tini
 COPY ./default.tmpl /templates/default.tmpl
 COPY ./alertmanager-bot /usr/bin/alertmanager-bot
 
-USER nobody
```

Comment:
not sure why this was added, but with this the folder inside can only be written with root so removing this allows running the cli as root to have write access